docs/pages/blog/mui-x-v7.md

```diff
@@ -0,0 +1,361 @@
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+```
+
+```jsx
+// date-fns v3.x
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFnsV3';
+```
+
+### Date Time Range Picker [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+Support new use cases with this most anticipated component.
+The latest addition to the Date and Time Pickers suite is the `<DateTimeRangePicker />`, a component designed to enrich the user experience with the advanced date and time selection.
+This intuitive picker simplifies the process of selecting date and time ranges, making it ideal for applications that require detailed scheduling or period selection.
+
+<video preload="metadata" autoplay muted loop playsinline width="600" height="510">
+  <source src="/static/blog/mui-x-v7/date-time-range-picker.mp4" type="video/mp4">
+</video>
+
+Learn how to use and customize the [new component](https://mui.com/x/react-date-pickers/date-time-range-picker/) now!
+
+## What's next
+
+As usual, we're committed to further improving the library by introducing new features, refining existing ones, and focusing on community feedback to shape our roadmap.
+
+Throughout this year, we'll invest in making MUI X components **design agnostic**. The components will still ship using Material UI as the default, but we're striving to ensure that they seamlessly integrate with any design system.
+The result is advanced components offering extreme flexibility and customization options, enabling a broader range of application styles without compromising the high-quality use cases and workflows that MUI X is known for.".
+
+Additionally, the following are the deliveries we're planning for the next few months.
+
+### Data Grid
+
+- [Improved Server side integration](https://mui.com/x/react-data-grid/server-side-data/)
+- [Column management panel with support for pivoting and other features](https://github.com/mui/mui-x/issues/5700)
+- [Pivoting](https://github.com/mui/mui-x/issues/214) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+### Tree View
+
+- [Inline node name editing](https://github.com/mui/mui-x/issues/10112)
+- [Checkbox selection for Simple- and RichTreeView](https://github.com/mui/mui-x/pull/11452)
+- [Drag and drop support for RichTreeView](https://github.com/mui/mui-x/issues/9686)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Charts
+
+- Improved label flexibility and customization
+- [Zoom and navigation](https://github.com/mui/mui-x/issues/12503)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+- [Heatmap](https://github.com/mui/mui-x/issues/7926)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+### Date and Time Pickers
+
+- [TimeRangePicker](https://github.com/mui/mui-x/issues/4460)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
```

Comment:
Could it be a good opportunity to help build some momentum for PigmentCSS by listing it as a goal for all components?